web/src/tools/testing/bulk_rendering/build.sh

```diff
@@ -14,6 +14,8 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 # This script runs from its own folder
 cd "$THIS_SCRIPT_PATH"
 
+BUNDLE_CMD="node $KEYMAN_ROOT/common/web/es-bundling/build/common-bundle.mjs"
```

Comment:
What's the reason for not including `web/common.inc.sh`? Then we wouldn't have to define `BUNDLE_CMD` here...